From 50ff279aaddfd91dc3cdcfd5b7aec3978e63da25 Mon Sep 17 00:00:00 2001
From: Brad Anderson <eco@gnuk.net>
Date: Wed, 12 Feb 2025 11:40:27 -0700
Subject: [PATCH] Include <chrono> explicitly in TraceyView

In Visual Studio 17.13 the STL team at Microsoft cleaned up internal includes of other public STL headers (most notably <chrono>) to increase build throughput for STL users, but now you need to include them yourself rather than rely on the indirect inclusion.
---
 profiler/src/profiler/TracyView.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/profiler/src/profiler/TracyView.hpp b/profiler/src/profiler/TracyView.hpp
index 80db05746a..6217fae8c9 100644
--- a/profiler/src/profiler/TracyView.hpp
+++ b/profiler/src/profiler/TracyView.hpp
@@ -3,6 +3,7 @@
 
 #include <array>
 #include <atomic>
+#include <chrono>
 #include <functional>
 #include <memory>
 #include <string>
